package.json

```diff
@@ -18,8 +18,8 @@
     "test:coverage-branch": "npm run test:unit -- --coverage && git diff origin/develop | diff-test-coverage -c ./coverage/lcov.info -t lcov --",
     "test:e2e:browserstack": "node tests/e2e/browserstack.runner.js -c  tests/e2e/browserstack.conf.js",
     "test:e2e:local": "kill-port 9515 && nightwatch --config ./tests/e2e/local-chrome.conf.js",
-    "test:e2e:build": "cross-env VUE_APP_E2E=true STARGATE=http://localhost:9070 RPC=http://localhost:26657 vue-cli-service build --mode production",
-    "test:e2e:serve": "kill-port 9080 && cross-env VUE_APP_E2E=true STARGATE=http://localhost:9070 RPC=http://localhost:26657 vue-cli-service serve --mode production --port 9080 --https",
+    "test:e2e:build": "cross-env VUE_APP_E2E=true VUE_APP_GRAPHQL_URL=https://backend.lunie.io/v1/graphql STARGATE=http://localhost:9070 RPC=http://localhost:26657 vue-cli-service build --mode production",
```

Comment:
This should be `VUE_APP_GRAPHQL_URL=http://localhost:8080/v1/graphql` as the e2e tests should be run in an isolated setup. Imagine testing delegations.

Reply:
But no api v1 is deployed with e2e CI setup at por 8080 isn't? When I change to VUE_APP_GRAPHQL_URL=http://localhost:8080/v1/graphql tests fail again :-(

Reply:
You are correct. happy with your solution then. Feel free to revert.